### PR TITLE
test: Holder mode BDD test and support for options

### LINKED ISSUE #320: Holder mode - BDD tests
BDD tests for https://github.com/trustbloc/edge-service/issues/316 #323

### LINKED ISSUE #337: Holder sign API - Support for options
Add support for options for sign API #323

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #320 
closes #337 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/vc-rest/docker-compose.yml [hunk 1]
```diff
@@ -61,6 +61,33 @@ services:
     networks:
       - couchdb_bdd_net
 
+  holder.vc.rest.example.com:
```
Comment (troyronda) [inline test/bdd/fixtures/vc-rest/docker-compose.yml:64]: we need to add this to sandbox and aws.
@fqutishat @rolsonquadras

File: test/bdd/pkg/vc/vc_steps.go [hunk 2]
```diff
@@ -320,7 +309,7 @@ func (e *Steps) createProfileAndPresentation(user, credential, did, privateKey,
 
 	ldpContext := &verifiable.LinkedDataProofContext{
 		SignatureType:           "Ed25519Signature2018",
-		SignatureRepresentation: getSignatureRepresentation("JWS"),
+		SignatureRepresentation: bddutil.GetSignatureRepresentation("JWS"),
```
Comment (troyronda) [inline test/bdd/pkg/vc/vc_steps.go:312]: Do we have tests for both `Ed25519Signature2018` and `JsonWebSignature2020`?

File: test/bdd/fixtures/vc-rest/docker-compose.yml (hunk 1 above)
Comment (rolsonquadras) [reply test/bdd/fixtures/vc-rest/docker-compose.yml:64]: Yes, I'm going to update sandbox next

File: test/bdd/pkg/vc/vc_steps.go (hunk 2 above)
Comment (rolsonquadras) [reply test/bdd/pkg/vc/vc_steps.go:312]: Yes - https://github.com/trustbloc/edge-service/pull/342/files#diff-b65ece61ea44b974d209834e4299072cR19-R20

**Pull request merged**